kornia/augmentation/_2d/intensity/planckian_jitter.py

```diff
@@ -0,0 +1,118 @@
+                                               [0.4320, 0.4644, 0.5681],
+                                               [0.4223, 0.4648, 0.5844],
+                                               [0.4135, 0.4651, 0.5990],
+                                               [0.4054, 0.4653, 0.6121],
+                                               [0.3980, 0.4654, 0.6239],
+                                               [0.3911, 0.4655, 0.6346],
+                                               [0.3847, 0.4656, 0.6444],
+                                               [0.3787, 0.4656, 0.6532],
+                                               [0.3732, 0.4656, 0.6613],
+                                               [0.3680, 0.4655, 0.6688],
+                                               [0.3632, 0.4655, 0.6756],
+                                               [0.3586, 0.4655, 0.6820],
+                                               [0.3544, 0.4654, 0.6878],
+                                               [0.3503, 0.4653, 0.6933]]),
+
+                    'CIED': torch.Tensor([[0.5829, 0.4421, 0.2288],
+                                          [0.5510, 0.4514, 0.2948],
+                                          [0.5246, 0.4576, 0.3488],
+                                          [0.5021, 0.4618, 0.3941],
+                                          [0.4826, 0.4646, 0.4325],
+                                          [0.4654, 0.4667, 0.4654],
+                                          [0.4502, 0.4681, 0.4938],
+                                          [0.4364, 0.4692, 0.5186],
+                                          [0.4240, 0.4700, 0.5403],
+                                          [0.4127, 0.4705, 0.5594],
+                                          [0.4023, 0.4709, 0.5763],
+                                          [0.3928, 0.4713, 0.5914],
+                                          [0.3839, 0.4715, 0.6049],
+                                          [0.3757, 0.4716, 0.6171],
+                                          [0.3681, 0.4717, 0.6281],
+                                          [0.3609, 0.4718, 0.6380],
+                                          [0.3543, 0.4719, 0.6472],
+                                          [0.3480, 0.4719, 0.6555],
+                                          [0.3421, 0.4719, 0.6631],
+                                          [0.3365, 0.4719, 0.6702],
+                                          [0.3313, 0.4719, 0.6766],
+                                          [0.3263, 0.4719, 0.6826],
+                                          [0.3217, 0.4719, 0.6882]])
+                    }
+
+
+class PlanckianJitter(IntensityAugmentationBase2D):
+    r"""Apply planckian jitter transformation to input tensor
+    This is physics based color augmentation, that creates realistic
+    variations in chromaticity, this can simulate the illumination
+    changes in the scene. This is based on https://arxiv.org/pdf/2202.07993.pdf
```

Comment:
You may use bibtex and citation in the doc. Check [this](https://github.com/kornia/kornia/blob/master/docs/source/references.bib)

Reply:
Thank you for the reference, this is resolved.